rep-0003.rst

```diff
@@ -180,6 +180,49 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2018)
```

Comment:
Since k-turtle will likely be a LTS release again the end date will be the same as the end date of 16.04.